docs/src/index.rst

```diff
@@ -1,7 +1,7 @@
 .. _iris_docs:
 
-Iris Documentation
-==================
+Iris |version|
+========================
```

Comment:
As we are using a variable for the text of the heading, the `====` line needs to be the same length or more else there will be a build warning (and thus error).